src/test/ui/macros/format-foreign.stderr

```diff
@@ -2,51 +2,50 @@ error: multiple unused formatting arguments
   --> $DIR/format-foreign.rs:12:5
    |
 12 |     println!("%.*3$s %s!/n", "Hello,", "World", 4);
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^--------^^-------^^-^^
+   |                              |         |        |
+   |                              |         |        unused
+   |                              |         unused
+   |                              unused
    |
-note: argument never used
-  --> $DIR/format-foreign.rs:12:30
-   |
-12 |     println!("%.*3$s %s!/n", "Hello,", "World", 4);
-   |                              ^^^^^^^^
-note: argument never used
-  --> $DIR/format-foreign.rs:12:40
-   |
-12 |     println!("%.*3$s %s!/n", "Hello,", "World", 4);
-   |                                        ^^^^^^^
-note: argument never used
-  --> $DIR/format-foreign.rs:12:49
-   |
-12 |     println!("%.*3$s %s!/n", "Hello,", "World", 4);
-   |                                                 ^
    = help: `%.*3$s` should be written as `{:.2$}`
    = help: `%s` should be written as `{}`
    = note: printf formatting not supported; see the documentation for `std::fmt`
    = note: this error originates in a macro outside of the current crate
 
 error: argument never used
-  --> $DIR/format-foreign.rs:13:29
+  --> $DIR/format-foreign.rs:13:5
    |
 13 |     println!("%1$*2$.*3$f", 123.456);
-   |                             ^^^^^^^
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^-------^^
+   |                             |
+   |                             unused
```

Comment:
The remaining three cases in this diff all ended up bigger! Can they be special cased back to the old way?!